src/docs/build.js

```diff
@@ -50,6 +50,14 @@ function writeDocs (output) {
     }))
 }
 
+function writeMdDocs (output) {
+  const docsPath = utils.getPathToDocs()
+  return fs.ensureDir(docsPath)
+    .then(() => {
+      fs.writeFileSync(docsPath + '/index.md', output)
```

Comment:
please use `path.join(docsPath, 'index.md')` to ensure the path is usable on all oses